CHSHGame/Tests.qs

```diff
@@ -73,25 +73,29 @@ namespace Quantum.Kata.CHSHGame {
     // ------------------------------------------------------
     @Test("QuantumSimulator")
     operation T22_AliceQuantum_Test () : Unit {
-        using (q = Qubit()) {
-            EqualityFactB(AliceQuantum(false, q), false, "|0⟩ not measured as false");
-            Reset(q);
-
-            // apply the Pauli X gate
-            X(q);
-            EqualityFactB(AliceQuantum(false, q), true, "|1⟩ not measured as true");
-            Reset(q);
-
-            // apply the Hadamard gate
-            H(q);
-            EqualityFactB(AliceQuantum(true, q), false, "|+⟩ is not measured as false");
-            Reset(q);
-
-            // apply the Pauli X and then the Hadamard gate
-            X(q);
-            H(q);
-            EqualityFactB(AliceQuantum(true, q), true, "|-⟩ is not measured as true");
-            Reset(q);
+        for (_ in 1 .. 4) {
+            // repeat 4 times since we are testing a measurement and wrong basis still might get
```

Comment:
I'm not sure what's the best way to estimate the probability of a false positive in one iteration, since the learner can come up with some creative solutions... I'd just say something like "repeat the test several times to reduce the probability of a false positive" without exact numbers.

Reply:
addressed